### PR TITLE
ipywidgets integration + slider for ssl wrapper plotter

### LINKED ISSUE #1912: Use an ipywidgets slider to scroll through frames to plot
### Description of the task

Scroll through frames to plot with an ipywidgets slider

### Acceptance criteria

- [ ] Integrate ipywidgets to jupyter environment
- [ ] Update the SSL_WrapperPacket plotter to use ColumnDataSource's to update data

### PR DESCRIPTION
**Pull request by jordanzliu**
### Description

ipywidgets integration + slider for ssl wrapper plotter

as it turns out columndatasources are required for plots to live update

### Testing Done

https://user-images.githubusercontent.com/41462969/106106771-e41b0a80-60fa-11eb-89d2-d500ae02583a.mp4

### Resolved Issues

resolves #1912 

### Review Checklist

**_It is the reviewers responsibility to also make sure every item here has been covered_**

- [ ] **Function & Class comments**: All function definitions (usually in the `.h` file) should have a javadoc style comment at the start of them. For examples, see the functions defined in `thunderbots/software/geom`. Similarly, all classes should have an associated Javadoc comment explaining the purpose of the class.
- [ ] **Remove all commented out code**
- [ ] **Remove extra print statements**: for example, those just used for testing
- [ ] **Resolve all TODO's**: All `TODO` (or similar) statements should either be completed or associated with a github issue
- [ ] **Justify drops in code coverage**: If this PR results in a non-trivial drop in code coverage (a bot should post a coverage diagram as a comment), please justify why we can't test the code that's not covered.

### CODE REVIEW

File: src/python_tools/notebooks/plot_ssl_wrapperpacket.ipynb [hunk 1]
```diff
@@ -3,21 +3,23 @@
   {
    "cell_type": "code",
    "execution_count": 1,
-   "id": "later-plasma",
+   "id": "common-serbia",
```
Comment (jonathanlew) [inline src/python_tools/notebooks/plot_ssl_wrapperpacket.ipynb:6]: just wondering what this is

File: src/python_tools/notebooks/plot_ssl_wrapperpacket.ipynb [hunk 2]
```diff
@@ -340,6 +342,14 @@
...
      "output_type": "display_data"
     },
+    {
+     "name": "stderr",
+     "output_type": "stream",
+     "text": [
+      "BokehUserWarning: ColumnDataSource's columns must be of the same length. Current lengths: ('robot_ids', 0), ('robot_ori_line_seg_xs', 1), ('robot_ori_line_seg_ys', 1), ('robot_xs', 0), ('robot_ys', 0)\n",
+      "BokehUserWarning: ColumnDataSource's columns must be of the same length. Current lengths: ('robot_ids', 0), ('robot_ori_line_seg_xs', 1), ('robot_ori_line_seg_ys', 1), ('robot_xs', 0), ('robot_ys', 0)\n"
+     ]
+    },
```
Comment (jonathanlew) [inline src/python_tools/notebooks/plot_ssl_wrapperpacket.ipynb:352]: is this an output?

**Review by jonathanlew — COMMENTED** [review]
very cool

**Review by BrianS99 — APPROVED** [review]
Looks good!

**Review by kelvinkoon — COMMENTED** [review]
LGTM, nothing to add besides existing comments 💯

File: src/python_tools/notebooks/plot_ssl_wrapperpacket.ipynb (hunk 1 above)
Comment (jordanzliu) [reply src/python_tools/notebooks/plot_ssl_wrapperpacket.ipynb:6]: its the output of bokeh plotting, this is not code that i changed or anything. notebooks will become easier to review after #1910 is resolved.

File: src/python_tools/notebooks/plot_ssl_wrapperpacket.ipynb (hunk 2 above)
Comment (jordanzliu) [reply src/python_tools/notebooks/plot_ssl_wrapperpacket.ipynb:352]: yeah

**Review by jonathanlew — APPROVED** [review]

**Review by akhilveeraghanta — APPROVED** [review]
:fire: works like a charm

**Pull request merged**